open-api/src/main/resources/yaml/openapi.yaml

```diff
@@ -0,0 +1,20 @@
+openapi: 3.0.3
+
+info:
+  title: Java-Home-Project-CQRS-Javelin-reader
```

Comment:
This open-api inplementation will be common for few modules, so you no need to specify reader module here.